mne/channels/montage.py

```diff
@@ -567,8 +571,25 @@ class DigMontage(object):
     Notes
     -----
     .. versionadded:: 0.9.0
+
     """
 
+    # XXX: To add to doc
+    #
+    # Dig points in DigMontage.dig should either be in known=['head', 'meg'] or
+    # 'unknown' if 'unknown' the digitization can contain (or not) fiducials
+    # also in 'unknown' coordinate system which is supposed to be the same.
```

Comment:
it's not so clear. With the ECoG  / iEEG  / sEEG use cases I anticipate that these values are in MRI coordinate.

Reply:
Agreed based on our IRL conversation but this should be tackled soon enough